src/modules/timepix3/dto/TimepixDataTypes.h

```diff
@@ -248,14 +248,14 @@ struct PixelReadout {
    * @param dCol The digital column of the pixel.
    * @param sPix The sub pixel index of the pixel.
    * @param pix The pixel index within the sub pixel.
+   * @param toa The Time-of-Arrival value of the pixel.
    * @param ToT The Time-over-Threshold value of the pixel.
    * @param fToA The fine Time-of-Arrival value of the pixel.
-   * @param toa The Time-of-Arrival value of the pixel.
    * @param spidrTime The SPIDR timestamp of the pixel.
    */
```

Comment:
Please remember to use the C++ comment style for Doxygen ;-)

/// \brief  etc.